types/awesome-notifications/index.d.ts

```diff
@@ -85,12 +85,27 @@ declare namespace AWN {
     }
 
     class Elem {
+        beforeInsert(): void;
+        afterInsert(): void;
         insert(): Elem;
+        replace(el?: Elem): Elem;
```

Comment:
```suggestion
        replace(el: Elem): Elem;
```
Is this parameter optional? [package code](https://github.com/f3oall/awesome-notifications/blob/2fd96e6215c7e14bfdcdecd23aea1864e5265c58/src/elem.js#L23)

Reply:
You are right. I changed it.